common/views/themes/config.ts

```diff
@@ -145,6 +145,12 @@ export const sizes = {
   headerLarge: 1040,
 };
 
+export const cardSizes = {
+  small: 0,
```

Comment:
these were finger-in-the-air sizes from changing my browser width and seeing where I thought it was looking less than good currently